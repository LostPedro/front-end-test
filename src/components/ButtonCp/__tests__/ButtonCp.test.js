import React from 'react'
import {render, cleanup} from '@testing-library/react'
import {ButtonCp} from '../'

afterEach(cleanup)

it('should take a snapshot', () => {
  const {asFragment} = render(<ButtonCp />)

  expect(asFragment(<ButtonCp />)).toMatchSnapshot()
})
