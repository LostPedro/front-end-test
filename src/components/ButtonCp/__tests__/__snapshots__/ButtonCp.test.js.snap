// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <button
    class="button-component"
    type="button"
  >
    <div />
  </button>
</DocumentFragment>
`;
